doc/Vagrant.md

```diff
@@ -0,0 +1,40 @@
+# Vagrant
+
+## Requirements
+
+* Vagrant
+* vagrant-host-manager plugin
+
+## Setup
+
+Boot Vagrant machine login and go to the project directory:
+
+```bash
+vagrant up
+vagrant ssh
+cd /vagrant
+```
+
+Install the dependencies:
+```bash
+composer install
+```
+
+Create database:
+
+```bash
+./vendor/bin/doctrine orm:schema-tool:create
+./vendor/bin/doctrine orm:generate-proxies
+```
+
+Copy prepared configuration:
+
+```bash
+
```

Comment:
Is this intended to be blank? I think you missed the copy command here from the docs.